engine/statmetrics.go

```diff
@@ -158,6 +166,22 @@ func (asr *StatASR) AddEvent(evID string, ev utils.DataProvider) (err error) {
 	return
 }
 
+func (asr *StatASR) OneEvent(ev utils.DataProvider) (err error) {
+	var answered int
+	if len(asr.Events) != 0 {
```

Comment:
We are not allowed to overwrite what we have already,better just take the first event from the queue. So also the asr.Events check against > 1